app/views/searches/new.html.erb

```diff
@@ -1,21 +1,11 @@
-<% content_for(:page_title_prefix, "Search") %>
+<% content_for(:page_title_prefix, "Find information on roles") %>
 
-<h2 class="heading-xlarge">Explore work you could do</h2>
+<h2 class="heading-xlarge">Find information on roles</h2>
 
 <% if current_scrapbook.saved_occupations.any? %>
```

Comment:
you are right. `ScrapbooksController` has very similar code and I simply assume that this is the corresponding controller. my bad. I'm not sure what is the ROR way of doing this.